### ISSUE TITLE
[BUG] Decompiling REWARDTABLE.MBIN creates an error

### ISSUE THREAD

**Issue by LeeC2202** [monkeyman192/MBINCompiler#115]
This is the first time I have used this software and the first time I have tried to mod NMS. Another MBIN file decompiled and then recompiled with no errors but this one doesn't want to play ball.

**Describe the bug**
Decompiling REWARDTABLE.MBIN using drag & drop causes an exception

**Expected behavior**
Expected file to be decompiled to REWARDTABLE.EXML

**Steps To Reproduce**
Drag & Drop REWARDTABLE.MBIN onto MBINCompiler.exe

**Specs (please complete the following information, if applicable):**
 - libMBIN v1.77.0.2
 - NMS v1.77
 - MBIN File: REWARDTABLE.MBIN
 - MBIN File Version: Unsure where to get this info

Log FIle:
[MBINCompiler.log](https://github.com/monkeyman192/MBINCompiler/files/2880484/MBINCompiler.log)

**Comment by monkeyman192** [comment]
@LeeC2202 Do you still have this problem?

**Comment by LeeC2202** [comment]
I've stopped playing NMS to be honest. I used to write script mods for GTA V but this file modding in NMS just wasn't my thing. Because of how the file structures work, it seems any file might need any version of mbin and I just got a bit tired of trying to match the pairs.

I probably should have explained that in updates to my bug reports, because I don't like wasting people's time... sorry about that. :(
